src/briefcase/integrations/windows_sdk.py

```diff
@@ -0,0 +1,136 @@
+        default_directories = [Path(r"C:\Program Files (x86)\Windows Kits\10")]
+
+        # Return user-specified SDK first
+        if environ_sdk := tools.os.environ.get("WindowsSdkDir"):
+            if environ_sdk_version := tools.os.environ.get("WindowsSDKVersion"):
+                yield environ_sdk, environ_sdk_version
+                raise BriefcaseCommandError(
+                    f"""\
+The 'WindowsSdkDir' and 'WindowsSDKVersion' environment variables do not point
+to a valid install of the Windows SDK v10:
+
+    WindowsSdkDir:     {environ_sdk}
+    WindowsSDKVersion: {environ_sdk_version}
+"""
+                )
+
+        seen_sdk_dirs = set()
+        for hkey in [winreg.HKEY_LOCAL_MACHINE, winreg.HKEY_CURRENT_USER]:
+            for access_right in access_right_precedence:
+                try:
+                    with winreg.OpenKeyEx(hkey, sdk_key, access=access_right) as key:
+                        sdk_dir, key_type = winreg.QueryValueEx(key, install_dir_subkey)
+                        if key_type != winreg.REG_SZ or not sdk_dir:
+                            continue
+
+                        sdk_dir = Path(sdk_dir)
+                        if sdk_dir in seen_sdk_dirs or not sdk_dir.is_dir():
+                            continue
+
+                        # Return the "latest" installed SDK first
+                        reg_version, key_type = winreg.QueryValueEx(key, version_subkey)
+                        if key_type == winreg.REG_SZ and reg_version:
+                            # the registry doesn't have the "servicing" version part
+                            reg_version += ".0"
+                            yield sdk_dir, reg_version
+
+                        # Return SDKs that may be installed at sdk_dir location
+                        inferred_versions = reversed(
+                            [d.name for d in (sdk_dir / "bin").glob("10.*.*.0/")]
+                        )
+                        for version in inferred_versions:
+                            if not version == reg_version:
+                                yield sdk_dir, version
+
+                        seen_sdk_dirs.add(sdk_dir)
+                except FileNotFoundError:
+                    pass  # ignore missing keys
+
+        for sdk_dir in default_directories:
+            if sdk_dir not in seen_sdk_dirs and sdk_dir.is_dir():
+                yield sdk_dir, None
+
+    @classmethod
+    def verify(cls, tools):
+        """Verify the Windows SDK installed with needed components.
+
+        :param tools: ToolCache of available tools
+        """
+        # short circuit since already verified and available
+        if hasattr(tools, "windows_sdk"):
+            return tools.windows_sdk
+
+        windows_sdk = None
+        for sdk_dir, sdk_version in cls._windows_sdks(tools):
+            # The code signing tool `signtool.exe` must exist
+            if not (sdk_dir / "bin" / sdk_version / "x64" / "signtool.exe").is_file():
```

Comment:
This looks like it might be a platform-specific path; we don't make any ARM guarantees for Windows at present, but by way of future proofing, it would be worth parameterising this.

Reply:
Parameterized.